.browserslistrc

```diff
@@ -0,0 +1,3 @@
+current node
+last 2 versions and > 2%
+ie > 10
```

Comment:
Somehow, we need to test these components on IE 11, to make sure they render correctly.